tests/test_dynamodb.py

```diff
@@ -48,8 +49,8 @@ class AliasKeyModel(BaseModel):
 
     @root_validator(pre=True)
     def type_from_typ(cls, values):
-        if 'typ' in values:
-            values['type'] = values.pop('typ')
+        if "typ" in values:
+            values["type"] = values.pop("typ")
```

Comment:
Is it possible to undo these formatting changes? They're not meaningful for the PR and pollute the git history.